samples/bluetooth/central_hr_coded/README.rst

```diff
@@ -0,0 +1,80 @@
+.. _bluetooth_central_hr_coded:
+
+Bluetooth: Central Heart Rate Monitor with Coded PHY
+####################################################
+
+Overview
+********
+
+Similar to the :ref:`zephyr:bluetooth_central_hr` sample, except that this application specifically looks for heart rate monitors using LE Coded PHY.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set23_start
+   :end-before: set23_end
```

Comment:
This block could be changed to:

```
.. table-from-rows:: /includes/sample_board_rows.txt
   :header: heading
   :rows: nrf5340pdk_nrf5340_cpuapp_and_cpuappns, nrf52840dongle_nrf52840, nrf52840dk_nrf52840
```

And then you could discard the doc/nrf/includes/boardname_tables/sample_boardnames.txt changes from the PR.

Reply:
@b-gent Since the table was added by you, do you agree with this?

Reply:
Yes, this is a very recent update to how we do these tables, I can change this tomorrow.